wandb/sdk/wandb_init.py

```diff
@@ -925,26 +931,29 @@ def _attach(
     if logger is None:
         raise UsageError("logger is not initialized")
 
-    manager = _wl._get_manager()
-    response = manager._inform_attach(attach_id=attach_id) if manager else None
-    if response is None:
+    service = _wl.service
+    if not service:
+        raise UsageError(f"Unable to attach to run {attach_id}")
```

Comment:
Maybe something like couldn't find the service and wasn't able to attach to run... ?